### ISSUE TITLE
Option to Auto-Send

### ISSUE THREAD

**Issue by AstudilloMarillo** [hlaueriksson/GEmojiSharp#30]
I'd like the ability to Auto-Send the emoji when selecting the one you want. It would save me having to do a paste action. Maybe not everyone will want to have this on, so a toggle would be best. Flow Launcher's 'Emoji+' plugin has a similar feature (although it doesn't work any more, atm).

**Comment by hlaueriksson** [comment]
Hi @AstudilloMarillo

When you say Auto-Send, do you mean type the selected emoji to the window that previously had focus?
Something like what the [Input Typer](https://github.com/CoreyHayward/PowerToys-Run-InputTyper) plugin does?

**Comment by AstudilloMarillo** [comment]
> When you say Auto-Send, do you mean type the selected emoji to the window that previously had focus?
> Something like what the [Input Typer](https://github.com/CoreyHayward/PowerToys-Run-InputTyper) plugin does?

Yes! Although I was initially thinking of auto-pasting to the window that previously had focus, but I guess typing is more robust.

**Comment by AstudilloMarillo** [comment]
any update on this? :)

**Comment by hlaueriksson** [comment]
Not yet. I released a new version, but didn't work on this issue.

If you feel up to it, you can implement this. PRs are welcome!

**Comment by hlaueriksson** [comment]
Hi @AstudilloMarillo

I just added support for updating the plugin from within PowerToys Run.

It will look something like this, but with a different version:

![ptrun](https://github.com/user-attachments/assets/430c9a9d-beb5-4c53-8e5f-ff746f3a0805)

I would like to use this issue to test the update feature.
So when the auto-send feature is added, that I will let you know.
Then you can install it with the included updater.

**Comment by AstudilloMarillo** [comment]
> I would like to use this issue to test the update feature.
> So when the auto-send feature is added, that I will let you know.
> Then you can install it with the included updater.

Sounds good!


> I just added support for updating the plugin from within PowerToys Run.

Wonderful feature btw, should be native to PowerToys Run honestly. The lack of version control on this program is pretty annoying.
